src/fides/api/ops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -1685,7 +1685,9 @@ def create_privacy_request_func(
                 detail=exc.args[0],
             )
         except Exception as exc:
-            logger.error("Exception: {}", Pii(str(exc)))
+            as_string = Pii(str(exc))
+            error_cls = str(exc.__class__)
```

Comment:
This will print as `<class 'ValueError'>`, we could make it print as just `ValueError` with `__name__`

```suggestion
            error_cls = str(exc.__class__.__name__)
```